src/main/js/components/editor/AddStepSelectionSheet.jsx

```diff
@@ -101,7 +101,7 @@ export class AddStepSelectionSheet extends Component<DefaultProps, Props, State>
         return (
             <div className="editor-step-selection-dialog">
                 <div className="editor-step-search">
-                    <Icon icon="search" style={{ fill: '#ddd' }} size={22} />
+                    <Icon icon="ActionSearch" color="#dddddd" size={22} />
```

Comment:
when the icon gets rendered it actually gets converted to inline css, so it is the same thing as before in this case